Firebase/Messaging/Public/FIRMessagingExtensionHelper.h

```diff
@@ -22,12 +22,14 @@
  __IOS_AVAILABLE(10.0)
 @interface FIRMessagingExtensionHelper : NSObject
 
- /// Call this API to complete your notification content modification. If you like to
+#if TARGET_OS_IOS
+/// Call this API to complete your notification content modification. If you like to
```

Comment:
Does this need __IOS_AVAILABLE(10.0)?

Reply:
Yes, this API is only for iOS 10 above because UNNotificationContent

Reply:
But I just realize macOS also has notification service extension so I will extend this to macOS support too.